client/network/test/Cargo.toml

```diff
@@ -13,6 +13,7 @@ repository = "https://github.com/paritytech/substrate/"
 targets = ["x86_64-unknown-linux-gnu"]
 
 [dependencies]
+async-std = "1.6.5"
```

Comment:
```suggestion
async-std = "1.7.0"
```

Reply:
Today we are using `async-std` `v1.6.5` throughout the entire Substrate codebase. This will eventually become `v1.7.0` (with the help of dependabot). I would rather update all usages of `async-std` at once, instead of a one-off update with this pull request.